src/api/LocalData/users.ts

```diff
@@ -0,0 +1,236 @@
+/* eslint-disable max-len */
+
+export const users = [
```

Comment:
You should receive this array from api

Reply:
fix this comment, you should fetch array with users from api